### PR TITLE
Adding new Problem

### PR DESCRIPTION
**Pull request by LoganHFink**
For assignment 3 of SF-1

### CODE REVIEW

File: first-a-then-b/io.json
Comment (ericabournival) [inline first-a-then-b/io.json:1]: I don't believe that you need the double square brackets for inputting a list

File: first-a-then-b/description.md
Comment (ericabournival) [inline first-a-then-b/description.md:1]: For your examples, do an underscore, I know it highlights it in red when doing it in vim, but if you put dashes, the programmer would get confused.

Comment (LoganHFink) [reply first-a-then-b/description.md:1]: Good idea, I changed it and pushed the new version so it should be up to date now

Comment (ericabournival) [reply first-a-then-b/description.md:1]: Perfect :)

**Comment by emaj9** [comment]
excellent

**Pull request merged**